template/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "My Moleculer-based microservices project",
   "scripts": {
     "build": "tsc --build tsconfig.json",
-    "dev": "ts-node ./node_modules/moleculer/bin/moleculer-runner.js --hot --repl --config moleculer.config.ts services/**/*.service.ts",
+    "dev": "ts-node ./node_modules/moleculer/bin/moleculer-runner.js --hot --repl --config moleculer.config.ts 'services/**/*.service.ts'",
```

Comment:
What operating system did you test on?

This seems to me to be something unique to the S.O.

Reply:
As shown in the screenshot, testing it in macos.

BTW, what is `S.O.` ?